src/ubx.h

```diff
@@ -1006,6 +1014,7 @@ class GPSDriverUBX : public GPSBaseStationSupport
 		u_blox9 = 9, ///< M9N, or any F9*, except F9P
 		u_blox9_F9P = 10, ///< F9P
 		u_blox10 = 11,
+		u_blox9_F9P_15B = 12, ///< ZED-F9P-15B
```

Comment:
These enums are ultimately for us, would something more descriptive like this work?

u_blox9_F9P -> u_blox9_F9P_L1L2
u_blox9_F9P_15B -> u_blox9_F9P_L1L5

Reply:
Done